src/browser/components/SettingsPage.jsx

```diff
@@ -418,6 +418,25 @@ export default class SettingsPage extends React.Component {
     });
   }
 
+  saveDownloadLocation = (location) => {
+    this.setState({
+      downloadLocation: location,
+    });
+    setImmediate(this.saveSetting, CONFIG_TYPE_APP_OPTIONS, {key: 'downloadLocation', data: location});
+  }
+
+  handleChangeDownloadLocation = (e) => {
+    this.saveDownloadLocation(e.target.value);
+  }
+
+  selectDownloadLocation = () => {
+    const message = 'Specify the folder where files will download';
+    remote.dialog.showOpenDialog({defaultPath: '',
```

Comment:
i don't remember if we had a problem with that part, but we should probably show either last configured or default location.

Reply:
@Willyfrog I wonder if we need to set this, since it's already interpreted from defaultSettings in main.js?